chore(release): bump version to 0.12.5

thsig opened the pull request.

eysi09 commented on docs/reference/providers/conftest-kubernetes.md:
```diff
@@ -14,7 +14,7 @@ Simply add this provider to your project configuration, and configure your polic
 modules.
 
-See the [conftest example project](https://github.com/garden-io/garden/tree/0.12.4/examples/conftest) for a simple
+See the [conftest example project](https://github.com/garden-io/garden/tree/v0.12.5/examples/conftest) for a simple
```
That `v` is not suppose to be there. See #2054. There are more cases like this.

thsig replied:
This should be fixed now.

thsig commented:
@eysi09: Looks like I forgot to update this PR and get it merged.

I've now rebased it on `master` and updated the generated doc links.

I also cleaned up a few hardcoded (but harmless) references to older versions of example projects in a couple of guides.
